src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.RuntimeOnly.cs

```diff
@@ -1936,6 +1936,20 @@ public static void XmlReflectionImporterTest()
         Assert.False(xmp.CheckSpecified);
     }
 
+    [Fact]    
+    public static void XmlSchemaExporter_ExportMembersMapping_NotSupportedDefaultValue()
```

Comment:
I know that "This API supports the product infrastructure and is not intended to be used directly from your code." but the api is public.